mod/host.py

```diff
@@ -1076,6 +1076,8 @@ def report_current_state(self, websocket):
 
             if self.transport_sync == "link":
                 self.set_link_enabled(True)
+            # if self.transport_sync == "midi_clock_slave":
+            #     self.set_midi_clock_slave_enabled(True)
```

Comment:
Why comments?

Reply:
not sure anymore why it's been commented
just removed the comments